### PR TITLE
DualUniverse

### PR DESCRIPTION
**Pull request by solomoncyj**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.1 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.1.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/70053)

### CODE REVIEW

File: manifests/n/Novaquark/DualUniverse/Beta 1/Novaquark.DualUniverse.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+PackageIdentifier: Novaquark.DualUniverse
+PackageVersion: Beta 1
+PackageLocale: en-US
+Publisher: Novaquark
+PackageName: Dual Universe
+License: proprietary
+Copyright: Novaquark
+ShortDescription: THE SPACE MMO ENTIRELY BUILT AND DRIVEN BY PLAYERS
+ManifestType: defaultLocale
+ManifestVersion: 1.2.0
```
Comment (OfficialEsco) [inline manifests/n/Novaquark/DualUniverse/Beta 1/Novaquark.DualUniverse.locale.en-US.yaml:13]:
```suggestion
ManifestVersion: 1.1.0
```

**Review by OfficialEsco — CHANGES_REQUESTED** [review]
VirtualBoxVM_Fsjlm2ZKTr

File: manifests/n/Novaquark/DualUniverse/Beta 1/Novaquark.DualUniverse.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: Novaquark.DualUniverse
+PackageVersion: Beta 1
+InstallerSwitches:
+  Custom: /SUPPRESSMSGBOXES
```
Comment (OfficialEsco) [inline manifests/n/Novaquark/DualUniverse/Beta 1/Novaquark.DualUniverse.installer.yaml:7]:
```suggestion
  Custom: /NORESTART
```

**Review by OfficialEsco — CHANGES_REQUESTED** [review]
Msg boxes can contain important information are there for a reason, /NORESTART only suppresses unexpected restarts.

**Review by russellbanks — CHANGES_REQUESTED** [review]

File: manifests/n/Novaquark/DualUniverse/Beta 1/Novaquark.DualUniverse.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: Novaquark.DualUniverse
+PackageVersion: Beta 1
+PackageLocale: en-US
+Publisher: Novaquark
+PackageName: Dual Universe
+License: proprietary
+Copyright: Novaquark
+ShortDescription: THE SPACE MMO ENTIRELY BUILT AND DRIVEN BY PLAYERS
```
[the remark this answers is not in the document]
Comment (OfficialEsco) [reply manifests/n/Novaquark/DualUniverse/Beta 1/Novaquark.DualUniverse.locale.en-US.yaml:11]: Without the spaces the validation will fail, also `|-` should keep any custom formats we use, `|` is just partially

```suggestion
ShortDescription: Dual Universe is a forward-thinking online game featuring a persistent and shared universe made with next-generation voxels.
Description: |-
  Dual Universe is a forward-thinking online game featuring a persistent and shared universe made with next-generation voxels.
  As a true precursor to the metaverse, the game world is fully editable, and the game experience is entirely driven by its players.
  Dual Universe is currently in public beta and reaching the final stretch toward launch.
Agreement:
  - AgreementLabel: EULA
    AgreementUrl: https://www.dualuniverse.game/legal/eula
```

File: manifests/n/Novaquark/DualUniverse/Beta 1/Novaquark.DualUniverse.installer.yaml [hunk 4]
```diff
@@ -0,0 +1,15 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: Novaquark.DualUniverse
+PackageVersion: Beta 1
+InstallerSwitches:
+  Custom: /NORESTART
+Installers:
+- Architecture: x86
```
Comment (OfficialEsco) [inline manifests/n/Novaquark/DualUniverse/Beta 1/Novaquark.DualUniverse.installer.yaml:9]:
```suggestion
- Architecture: x64
```

**Review by OfficialEsco — CHANGES_REQUESTED** [review]
InstallationVerification-x86-Scope_Undefined-Locale_Undefined-ErrorScreenshot_20220815152634

File: manifests/n/Novaquark/DualUniverse/Beta 1/Novaquark.DualUniverse.locale.en-US.yaml [hunk 5]
```diff
@@ -0,0 +1,24 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: Novaquark.DualUniverse
+PackageVersion: Beta 1
+PackageLocale: en-US
+Publisher: Novaquark
+PublisherURL: https://www.novaquark.com
```
Comment (OfficialEsco) [inline manifests/n/Novaquark/DualUniverse/Beta 1/Novaquark.DualUniverse.locale.en-US.yaml:8]: > Manifest Warning: Unknown field. Field: Agreement Manifest Error: All field names should be PascalCased. Field: PublisherURL

```suggestion
PublisherUrl: https://www.novaquark.com
```

**Review by OfficialEsco — CHANGES_REQUESTED** [review]

**Review by OfficialEsco — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**